packages/app-store/salesforce/lib/CrmService.ts

```diff
@@ -497,32 +477,42 @@ export default class SalesforceCRMService implements CRM {
         appOptions.createEventOnLeadCheckForContact &&
         !forRoundRobinSkip
       ) {
-        // Get any matching contacts
-        const contactSearch = await conn.query(
```

Comment:
Same here, we were making one call to check for a lead and then a second one to check for a contact.